packages/vscode/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "devDependencies": {
     "@types/tar-stream": "^1.6.0",
+    "cross-env": "^5.2.0",
```

Comment:
@kylecarbs why wasn't this here already?

Reply:
I've seen a commit yesterday, which removed the global instal of this packet from travis.
I think you guys have this package installed globally :)

Reply:
This should be fixed now; the scripts directly reference the `cross-env` binary installed in the root `node_modules` directory.